style.css

```diff
@@ -313,3 +318,216 @@ textarea {
   color: #42526e;
   margin-bottom: 147px;
 }
+
+/* -----------------------------------media */
+
+@media screen and (min-width: 1080px) {
+  body {
+    background-image: url(imege/Header-llustration-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-size: 100%;
+  }
```

Comment:
- [ ] Please make all changes as requested by the previous reviewer, your actual design looks a bit not aligned properly, let's dive into the full design and make some improvement 👍🏻 
1. - [ ] Kindly add the arrow icon as requested by the previous review, seeing this image below, we have that icon in both mobile and desktop versions, you can add remaining icons to the social media icon too
<img width="1221" alt="icon" src="https://user-images.githubusercontent.com/79058364/160106988-27c3d732-dd77-4a4a-94dd-25a77896ca76.png">

### IT SHOULD BE LIKE THIS INSTEAD

<img width="745" alt="icon fig" src="https://user-images.githubusercontent.com/79058364/160107155-0cc9d436-22f0-43ff-81b6-e7437e4ec95f.png">


2. - [ ] As requested by the first reviewer, you should `margin: 0 auto;` to center your cards, please fix that, as for now it is not aligned yet, and please make some spaces between them as per the Figma design
<img width="1260" alt="work" src="https://user-images.githubusercontent.com/79058364/160106665-15c2c207-dede-43d1-b23d-a1a1b1b9486c.png">

## IT SHOULD BE LIKE THIS INSTEAD
<img width="708" alt="work fig" src="https://user-images.githubusercontent.com/79058364/160108081-03193bcf-238a-46cc-b8ae-f54f420fa6b0.png">

3. - [ ] The about section need some adjustments, please make it look like the original design by implementing changes as shown below

<img width="944" alt="form" src="https://user-images.githubusercontent.com/79058364/160109176-ced5b54b-97a5-45d9-a639-9db0d2545109.png">

- [ ] MAKE IT BE LIKE ⬇️ 

<img width="647" alt="form fig" src="https://user-images.githubusercontent.com/79058364/160109373-2156471f-8d7b-4fec-9fca-6352422a3317.png">

- [ ] Please add 2 more social media icons to your footer, I think there are some missing, isn't it? alright, please fix that.

<img width="993" alt="footer fig" src="https://user-images.githubusercontent.com/79058364/160109747-c6fe8555-6b31-4a85-9238-97ee6b277c1c.png">

- [ ] ORIGINAL DESIGN
<img width="740" alt="footer" src="https://user-images.githubusercontent.com/79058364/160109906-ac5c841f-bc18-41b2-b4ea-11d6be277291.png">

Reply:
Please check again the Figma design are assigned the desktop design to 1440px and you sent me a screenshot from 1306px and asked me to follow the rules.!!
besides social icons is not a crucial reason to stop my PR approval, there is no code reviewer asked me to change it before

Reply:
@BenMukebo kindly check my comment above

Reply:
Hello @MuhmmdUsama, let me check it out, but your desktop version media query is set to 1080px to say that the dimensions between 1080 > 1440px should return the same result. we should be able to see the correct design from 1080 and above which is not the case with your application. please hold on while am taking a look at it.

Reply:
Hey, am back, take a look at the gif I sent on slack because it is too big here , your web app is only working on size 1440px but you have set your media query to 1080px, either 1300 is breaking your design,

Reply:
Hi @BenMukebo  I already fixed the issues right now, could you please take another look